docs/connectors/postgresql.asciidoc

```diff
@@ -1176,8 +1176,8 @@ The following configuration properties are _required_ unless a default value is
 | Specifies how the connector should handle values for `DECIMAL` and `NUMERIC` columns: `precise` (the default) represents them precisely using `java.math.BigDecimal` values represented in change events in a binary form; or `double` represents them using `double` values, which may result in a loss of precision but will be far easier to use. `string` option encodes values as formatted string which is easy to consume but a semantic information about the real type is lost. See <<decimal-values>>.
 
 |`database.sslmode`
-|`disabled`
-|Whether to use an encrypted connection to the PostgreSQL server. Options include: *disabled* (the default) to use an unencrypted connection ; *required* to use a secure (encrypted) connection, and fail if one cannot be established; *verify_ca* like `required` but additionally verify the server TLS certificate against the configured Certificate Authority (CA) certificates, or fail if no valid matching CA certificates are found; *verify_full* like `verify_ca` but additionally verify that the server certificate matches the host to which the connection is attempted. See https://www.postgresql.org/docs/9.6/static/libpq-connect.html[the PostgreSQL documentation] for more information.
+|`disable`
+|Whether to use an encrypted connection to the PostgreSQL server. Options include: *disable* (the default) to use an unencrypted connection ; *require* to use a secure (encrypted) connection, and fail if one cannot be established; *verify-ca* like `require` but additionally verify the server TLS certificate against the configured Certificate Authority (CA) certificates, or fail if no valid matching CA certificates are found; *verify-full* like `verify-ca` but additionally verify that the server certificate matches the host to which the connection is attempted. See https://www.postgresql.org/docs/9.6/static/libpq-connect.html[the PostgreSQL documentation] for more information.
```

Comment:
Good catch, there's something incorrect indeed.

Looking at the [code](https://github.com/debezium/debezium/blob/master/debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java#L181), the values should be `disable`, `require`, `verify_ca` and `verify_full`, though (underscore instead of dash).

While at it, could you also adapt the description of the values in `PostgresConnectorConfig#SSL_MODE`?

Thanks a lot!

Reply:
I tried using underscore instead of dash, but that didn't work:
```
2018-06-05T18:53:30.951317989Z Jun 05, 2018 6:53:30 PM org.postgresql.Driver connect
2018-06-05T18:53:30.951343371Z SEVERE: Connection error: 
2018-06-05T18:53:30.951347673Z org.postgresql.util.PSQLException: Invalid sslmode value: verify_ca
2018-06-05T18:53:30.951350764Z  at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:101)
2018-06-05T18:53:30.951363932Z  at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)
2018-06-05T18:53:30.951367240Z  at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:194)
2018-06-05T18:53:30.951370691Z  at org.postgresql.Driver.makeConnection(Driver.java:431)
2018-06-05T18:53:30.951373609Z  at org.postgresql.Driver.connect(Driver.java:247)
2018-06-05T18:53:30.951376385Z  at io.debezium.jdbc.JdbcConnection.lambda$patternBasedFactory$1(JdbcConnection.java:161)
2018-06-05T18:53:30.951379313Z  at io.debezium.jdbc.JdbcConnection.connection(JdbcConnection.java:585)
2018-06-05T18:53:30.951382538Z  at io.debezium.connector.postgresql.connection.PostgresConnection.<init>(PostgresConnection.java:65)
2018-06-05T18:53:30.951385965Z  at io.debezium.connector.postgresql.PostgresConnector.validate(PostgresConnector.java:95)
2018-06-05T18:53:30.951388873Z  at org.apache.kafka.connect.runtime.AbstractHerder.validateConnectorConfig(AbstractHerder.java:265)
2018-06-05T18:53:30.951391666Z  at org.apache.kafka.connect.runtime.distributed.DistributedHerder$6.call(DistributedHerder.java:528)
2018-06-05T18:53:30.951394569Z  at org.apache.kafka.connect.runtime.distributed.DistributedHerder$6.call(DistributedHerder.java:525)
2018-06-05T18:53:30.951397341Z  at org.apache.kafka.connect.runtime.distributed.DistributedHerder.tick(DistributedHerder.java:261)
2018-06-05T18:53:30.951400373Z  at org.apache.kafka.connect.runtime.distributed.DistributedHerder.run(DistributedHerder.java:210)
2018-06-05T18:53:30.951403269Z  at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
2018-06-05T18:53:30.951406036Z  at java.util.concurrent.FutureTask.run(FutureTask.java:266)
2018-06-05T18:53:30.951408815Z  at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
2018-06-05T18:53:30.951411642Z  at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
2018-06-05T18:53:30.951414498Z  at java.lang.Thread.run(Thread.java:748)
2018-06-05T18:53:30.951417234Z 
2018-06-05T18:53:30.951942282Z 2018-06-05 18:53:30,951 INFO   ||  172.18.0.11 - - [05/Jun/2018:18:53:30 +0000] "POST /connectors HTTP/1.1" 500 64  3   [org.apache.kafka.connect.runtime.rest.RestServer]
```

I found what I think are the correct values in the JDBC driver docs: https://jdbc.postgresql.org/documentation/head/connect.html